### ISSUE TITLE
Add expand property to BottomSheet library

### ISSUE THREAD

**Issue by devnunu** [holixfactory/bottomsheetdialog-compose#5]
hello. I'm using your library very well. 
While using the library, I found that there is no expand function.
So, when I want a bottomSheet that covers the entire screen, I have to open the bottomSheet twice by swiping.

Can you fix this by adding a property for expand? If this property is added, the usability of the library will be better.
(By the way, your library is really cool)

**Comment by workspace** [comment]
Hi @devnunu. Thank you for your opinion and sorry for late reply. I'll check it soon.

**Comment by workspace** [comment]
Hi @devnunu, sorry for late reply.

I'll added some properties to set bottomsheet's behavior.
You can set BottomSheetBehavior.state and BottomSheetDialog will be expanded at first like this. Is it enough?

https://user-images.githubusercontent.com/7759511/198026596-4ff32bbf-3c2c-4566-ae7b-02e6fd6e61c5.mp4

**Comment by workspace** [comment]
@devnunu 
Closed with #8.

Try this
```
BottomSheetProperties(
    ...,
    behaviorProperties = BottomSheetBehavior(
        ...,
        state = State.Expanded
    ) 
)
```

**Comment by devnunu** [comment]
I'm also sorry for the late reply. Perfectly the feature I wanted. Thanks for taking care of it. you guys are the best team!
